src/Avalonia.Visuals/Platform/IDrawingContextImpl.cs

```diff
@@ -63,7 +63,7 @@ public interface IDrawingContextImpl : IDisposable
         /// <param name="brush">The brush used to fill the rectangle, or <c>null</c> for no fill.</param>
         /// <param name="pen">The pen used to stroke the rectangle, or <c>null</c> for no stroke.</param>
         /// <param name="rect">The rectangle bounds.</param>
-        /// <param name="boxShadows">Box shadow effect parameters</param>
+        /// <param name="boxShadow">Box shadow effect parameters</param>
```

Comment:
I think in this case the actual parameter name should be changed to `boxShadows` to be the same as in `RectangeNode` for example.